web/_js/draw.js

```diff
@@ -384,6 +402,65 @@ function initDraw(){
 		}
 	}
 
+	const getEntry = id => {
+		const entries = atlas.filter(entry => entry.id === id)
+		if (entries.length === 1) return entries[0]
+		return {}
+	}
+
+	let params = new URLSearchParams(document.location.search)
+
+	if (params.has('id')) {
```

Comment:
This doesn't work together with the new hashes. See https://github.com/fabi321/atlas/commit/391fbf179353051894929391132ca3c5017aeb93 for a fix (I can't pr on your repo)

Reply:
I have fixed this but I opt to use the params for now, just so it will always guranteed to start clean when the ID changes.